config/manager/manager.yaml

```diff
@@ -38,7 +38,7 @@ spec:
         - /manager
         args:
         - --leader-elect
-        image: 1password/onepassword-operator:latest
+        image: controller:latest
```

Comment:
Is there a way to name this similar to the image we will publish on DockerHub?

Reply:
I tried to set the same name, but it doesn't work with it. Need more time to investigate what else to be done to make it work.

Reply:
Just looked in the makefile and found this:
```
IMG ?= controller:latest
```

Maybe changing this should do the trick?

Reply:
Yeah, that works! Thanks! 😄